app/commands/training_data/code_tags_sample/normalize_tags.rb

```diff
@@ -0,0 +1,205 @@
+class TrainingData::CodeTagsSample::NormalizeTags
+  include Mandate
+
+  initialize_with :tags
+
+  def call
+    return nil if tags.nil?
+
+    tags.flat_map { |tag| MAPPING.fetch(tag, tag) }.uniq
```

Comment:
This not only normalizes the concept via the below mapping, but also removes any duplicates (we didn't do that before).